include/rtl_433_devices.h

```diff
@@ -12,7 +12,6 @@
 		DECL(elv_em1000) \
 		DECL(elv_ws2000) \
 		DECL(lacrossetx) \
-		DECL(acurite5n1) \
```

Comment:
do we have a "special" null device type, e.g deleted, which we can register here as a placeholder not to mess up the index ordering?

I _could_ leave this driver alone (I recall writing much of the initial driver, although it has been modified) in here, but its both duplicative and broken. Moving to a single one if it covers these use cases is probably better - but it's an interesting question as to the right approach.